packages/remix-forms/src/createForm.tsx

```diff
@@ -257,6 +256,7 @@ function createForm({
         radioWrapperComponent,
         fieldErrorsComponent,
         Error,
+        form.register,
```

Comment:
This dependency was missing, this might result in an runtime change.
In my tests it didn't change the render count of forms with `<Field` component.

Reply:
I don't care too much for this linter rule but since it can help us to not miss important things adding form.register as a dependency is a small price to pay.